libpod/sql_state_test.go

```diff
@@ -0,0 +1,520 @@
+package libpod
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/stretchr/testify/assert"
+)
+
+func getTestContainer(id, name string) *Container {
+	ctr := &Container{
+		config: &containerConfig{
+			ID:              id,
+			Name:            name,
+			RootfsImageID:   id,
+			RootfsImageName: "testimg",
+			UseImageConfig:  true,
+			StaticDir:       "/var/lib/libpod/",
```

Comment:
Lets keep this in containers /var/lib/containers/libpod?

Reply:
Actually probably should use tmpdirs for these.

Reply:
This isn't actually used, it's just temporary paths to check that the container is being retrieved correctly. Let me make that clear